ci/script.sh

```diff
@@ -37,10 +37,10 @@ main() {
     mkdir -p ci/builds
 
     if [ $T = x86_64-unknown-linux-gnu ]; then
-        if [ $TRAVIS_RUST_VERSION = nightly ]; then
-            # compile-fail tests
-            cargo test --test single --target $T
+        # compile-fail tests
+        cargo test --test single --target $T
```

Comment:
The compiler output might change between versions, so running this on all channels might cause issues later on

Reply:
good point. I have changed the condition to test only when `rustc --version == $MSRV`. $MSRV remains constant for a whole major version of rtfm (e.g. 1.36.0 for v0.5.x)